app/routes.js

```diff
@@ -1,7 +1,18 @@
-// const controller = require('./controllers/controller');
+const indexController = require('./controllers/index'),
+  userController = require('./controllers/users'),
+  generics = require('./middlewares/generics');
 
 exports.init = app => {
-  // app.get('/endpoint/get/path', [], controller.methodGET);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
-  // app.post('/endpoint/post/path', [], controller.methodPOST);
+  app.use(generics.logRequestInformation);
+
+  app.get('/', indexController.home);
+
+  app.post(
+    '/users',
+    userController.checkIfObligatoryParametersWereReceived,
```

Comment:
all of this should be done in a single function inside of createUser, its fine to divide the validation logic in different functions but there is no need for them to be middlewares.